docs/user-tutorial/data-diagnosis.md

```diff
@@ -0,0 +1,134 @@
+---
+id: data-diagnosis
+---
+
+# Data Diagnosis
+
+## Introduction
+
+This tool is to validate and diagnose the SuperBench benchmarking results for hundreds or thousands of servers automatically as to whether they meet the hardware performance/quality standard according to the baseline of metrics in the benchmarks,  and finally provides a list of failed servers with detailed information.
```

Comment:
`according to the baseline` => `comparing with the baseline`.
`,  and finally ` => one more whitespace after the `,`

Reply:
updated